pkg/views/workspace/create/view.go

```diff
@@ -179,8 +184,46 @@ func validateRepoUrl(repoUrl string, apiClient *apiclient.APIClient) (*apiclient
 		Url: result,
 	}).Execute()
 	if err != nil {
-		return nil, errors.New("Failed to fetch repository information. Please check the URL and try again.")
+		wrappedErr := "Failed to fetch repository information. Please check the URL and try again."
+		return nil, errors.New(WrapText(wrappedErr, GetTerminalWidth()))
 	}
 
 	return repo, nil
 }
+
+func WrapText(text string, width int) string {
```

Comment:
I suggest moving this function to some common utility because it can be used in more scenarios than just here.

I suggest `pkg/views/util`

Reply:
- Moved the function to the `help.go` in `pkg/views/util`